crates/router/src/core/routing.rs

```diff
@@ -818,6 +817,37 @@ pub async fn retrieve_linked_routing_config(
     }
 }
 
+pub async fn create_connector_agnostic_mandate_config(
```

Comment:
Nit:
```suggestion
pub async fn create_connector_agnostic_mandate_config(
```
pub async fn upsert_connector_agnostic_mandate_config(